src/server/components/comparison.ts

```diff
@@ -125,12 +125,22 @@ function getComparisonGene(gene: Gene, teams: Team[], scores: Scores[], allBiodo
     target_risk_score: geneScores ? geneScores.target_risk_score : null,
     genetics_score: geneScores ? geneScores.genetics_score: null,
     multi_omics_score: geneScores ? geneScores.multi_omics_score : null,
-    biodomains: geneBiodomains
+    biodomains: geneBiodomains,
+    target_enabling_resources: getTargetEnablingResources(gene)
   };
 
   return data;
 }
 
+export function getTargetEnablingResources(gene: Gene) {
```

Comment:
incoming data is booleans but we need a string array for filtering, so this function will create the needed string array.